python_base/Dockerfile

```diff
@@ -19,42 +19,40 @@
 # In applying this licence, CERN does not waive the privileges and immunities
 # granted to it by virtue of its status as an Intergovernmental Organization
 # or submit itself to any jurisdiction.
-FROM python:3.5
+FROM centos
 
-RUN apt-get update --fix-missing && \
-    apt-get install -y \
-      python-virtualenv \
-      python-dev \
-      git \
-      nodejs \
-      npm \
-      libpq-dev \
-      build-essential \
-      libxml2-dev \
-      libxslt1-dev \
-      libjpeg-dev \
-      libfreetype6-dev \
-      libtiff-dev \
-      libmagickwand-dev \
-      imagemagick \
-      ghostscript \
-      xpdf \
-      libffi-dev \
-      liblzma-dev && \
-    ln -s /usr/bin/nodejs /usr/bin/node && \
-    npm install -g \
-      npm \
-      node-sass \
-      clean-css \
-      requirejs \
-      uglify-js
+RUN yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
```

Comment:
I think that's only really useful (saves at least 1MB) if your commands do actually remove stuff from the layer, if not, the only extra space it will take is the metadata for that layer, just tested to collapse a couple of them, this is the result, before:

e09ae36cecf3        25 minutes ago      /bin/sh -c npm install -g         node-sass     25.66 MB            
cea865b140c7        27 minutes ago      /bin/sh -c npm install -g npm                   30.73 MB            
d97f7182da0e        27 minutes ago      /bin/sh -c yum install -y         ImageMagick   274 MB  

that's ~330.9MB, once collapsed:
744fbab9bf71        9 minutes ago       /bin/sh -c yum install -y         ImageMagick   330.1 MB            

So the overhead per-layer seems to be ~0.45MB (a lot imo...), but not that helpful compared to the full size.